src/app/scripts/cac/pages/cac-pages-map.js

```diff
@@ -57,8 +61,9 @@ CAC.Pages.Map = (function ($, Handlebars, _, MapControl, Routing, MockDestinatio
 
     return Map;
 
-    function showOptions() {
-        var moreOpt = '.sidebar-options .more-options';
+    function showOptions(event) {
+        var parent = $(event.target).parent().parent();
```

Comment:
I tend to strongly avoid selections like this, as it is extremely brittle, and I've been bitten by it before. At least with an id/class selector of some kind, you avoid breaking the selection if you move the dom tree around.